worker.go

```diff
@@ -385,7 +388,13 @@ func NewSpanWorker(sinks []sinks.SpanSink, cl *trace.Client, spanChan <-chan *ss
 // Work will start the SpanWorker listening for spans.
 // This function will never return.
 func (tw *SpanWorker) Work() {
+	capcmp := cap(tw.SpanChan) - 1
```

Comment:
Is this expensive?

Reply:
no, it's more about wanting to have a fixed number so that there's the minimum possible number of operations between the channel receive return and the conditional comparison operation, below. there's no guarantee of CPU instruction ordering, of course, but i think this gives us a better chance of not missing that we hit the cap.

Reply:
I was only asking since it's in a hot path. Godspeed!

Reply:
there's also like three more reasons it's unlikely to make a difference (which i could put in a comment), but it doesn't really hurt, soooo, eh :)

Reply:
oh, actually, it's not in the hotpath - it's outside the loop. that was the goal 😄